<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NewWorld Producer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Orders of:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelOrdersDate">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Order count:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelOrderCount">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelNom">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Surname:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPrenom">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelIdentifiant_lab">
        <property name="text">
         <string>Producer nÂ°:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelIdentifiant">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <widget class="QWidget" name="tabPrepa">
       <attribute name="icon">
        <iconset resource="prod.qrc">
         <normaloff>:/icones/commandes</normaloff>:/icones/commandes</iconset>
       </attribute>
       <attribute name="title">
        <string>Orders to prepare</string>
       </attribute>
       <attribute name="toolTip">
        <string>you'll find there orders</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableWidget" name="tableWidgetLDC">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="columnCount">
           <number>8</number>
          </property>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Checked</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>At</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Where</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>For name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>For Surname</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>What</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>HowMany</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Unity</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDelivery">
       <attribute name="icon">
        <iconset resource="prod.qrc">
         <normaloff>:/icones/delivery</normaloff>:/icones/delivery</iconset>
       </attribute>
       <attribute name="title">
        <string>Delivery</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Sale Point:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxSalePoint">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidgetLDC_2">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="columnCount">
           <number>6</number>
          </property>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Delivered</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>For name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>For Surname</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>What</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>HowMany</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Unity</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonQuit">
        <property name="text">
         <string>&amp;Quitter</string>
        </property>
        <property name="icon">
         <iconset resource="prod.qrc">
          <normaloff>:/icones/quit</normaloff>:/icones/quit</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="prod.qrc"/>
 </resources>
 <connections/>
</ui>
